test/examples/ucp_hello_world.c

```diff
@@ -329,6 +332,14 @@ static int run_ucx_server(ucp_worker_h ucp_worker)
         goto err;
     }
 
+    test_str = (char*) calloc(test_string_length + 1, sizeof(char));
```

Comment:
@guy101 I meant, the user needs 256 bytes to transfer. It means, we have to provide `255` chars + `0`